go.mod

```diff
@@ -17,3 +17,12 @@ require (
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+require (
```

Comment:
Why is there another require in here?

Reply:
I had to run `go mod tidy`. Otherwise it will neither compile nor run the tests. I also tried it within a Docker container. Same problem.
A standalone issue/PR could be more suitable.

Reply:
It's just run `go mod tidy` again, isn't it @olsibakullari? I've done it on your branch and this `require` has been gone after run it.

Reply:
I have removed the changes from this file as this is another issue. I think your testing it with Go Version `1.16` or lower. I am running `1.17`. In order to reproduce it try running `docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang:1.17 make test`.

Reply:
Yeah, I'm using go 1.16 on my machine and that was the problem